applications/nrf_desktop/configuration/nrf54h20dk_nrf54h20_cpuapp/app_dongle.overlay

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright (c) 2025 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include "app_common.dtsi"
+
+/ {
+	hid_dev_0: hid_dev_0 {
+		compatible = "zephyr,hid-device";
+		interface-name = "HID0";
+		protocol-code = "keyboard";
+		in-polling-period-us = <125>;
```

Comment:
Shouldn't we set it to 1000? We won't get 8kHz via LLPM anyway so that setting will only generate unnecessary NAKs.

Reply:
Using shorter USB polling intervals could reduce overall latency of HID reports from peripherals. NAKs should not cause problems. Keep in mind that a HID peripheral connected over USB in real-life scenario would not provide a HID report every 125 us (we tried to avoid NAKs in scenario with simulated motion that was constantly generated).